common/web/gesture-recognizer/src/engine/headless/gestureSource.ts

```diff
@@ -167,25 +203,61 @@ export class GestureSourceSubview<HoveredItemType> extends GestureSource<Hovered
    * @param initialHoveredItem  The initiating event's original target element
    * @param isFromTouch    `true` if sourced from a `TouchEvent`; `false` otherwise.
    */
-  constructor(source: GestureSource<HoveredItemType>, startAtEnd: boolean, preserveBaseItem: boolean) {
-    super(source.rawIdentifier, source.isFromTouch);
+  constructor(
+    source: GestureSource<HoveredItemType>,
+    configStack: typeof GestureSource.prototype['recognizerConfigStack'],
+    startAtEnd: boolean,
+    preserveBaseItem: boolean
+  ) {
+    let mayUpdate = true;
+    let start = 0;
+    let length = source.path.coords.length;
+    if(source instanceof GestureSourceSubview) {
+      const expectedLength = source._baseStartIndex + source.path.coords.length;
+      start = source._baseStartIndex;
```

Comment:
Given we've just declared it above, I think this is also clearer being in the same order:

```suggestion
      start = source._baseStartIndex;
      const expectedLength = start + length;
```